client/lib/index.js

```diff
@@ -330,11 +329,15 @@ module.exports = class Client extends PassThrough {
 								this.write(data);
 								process.send({ type, data });
 								break;
+							case 'status':
+								if (!data.success) {
+									process.exitCode = 2;
```

Comment:
Now we get the status from the remote device and set exit code to non-zero if the test fails.